lmms_eval/api/task.py

```diff
@@ -1023,6 +1024,15 @@ def concat_tar_parts(tar_parts, output_tar):
             download_config=download_config,
             **dataset_kwargs if dataset_kwargs is not None else {},
         )
+        
+        selected_datasets = DatasetDict()
+        if 'mmerealworld_cn' in self.task_name:
```

Comment:
Kind of hardcode here for selecting dataset, is it possible that you can split the dataset in different subset or split?

Reply:
I have split the dataset into different subsets.